workflow/rules/common.smk

```diff
@@ -120,7 +129,7 @@ def get_bioc_pkg_path(wildcards):
 
 def kallisto_params(wildcards, input):
     extra = config["params"]["kallisto"]
-    if len(input.fq) == 1:
+    if len(input.fq) == 1 or config["experiment"]["3-prime-rna-seq"]["activate"]:
```

Comment:
With 3-prime, samples should be single end, right? Hence, the "or" case is not required here.

Reply:
Noted, Changed